### PR TITLE
Added TPHere and updated translations.yaml

### PR DESCRIPTION
**Pull request by Paradox304**

### CODE REVIEW

File: NewEssentials/translations.yaml [hunk 1]
```diff
@@ -134,4 +134,7 @@ god:
   god: "You are now a God!"
   god_other: "{Player} is now a God!"
   mortal: "You are now a mortal!"
-  mortal_other: "{Player} has reappeared!"
\ No newline at end of file
+  mortal_other: "{Player} has reappeared!"
+  
+tphere:
+  successfull_tp: "{Player} has been tped to you"
```
Comment (archie426) [inline NewEssentials/translations.yaml:140]: Remove the l from successful (sorry I know this is petty but better than changing it later)

File: NewEssentials/Commands/CTPHere.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        public CTPHere(IServiceProvider serviceProvider, IStringLocalizer stringLocalizer, IUserManager userManager) :base(serviceProvider)
+        {
+            m_ServiceProvider = serviceProvider;
+            m_StringLocalizer = stringLocalizer;
+            m_UserManager = userManager;
+        }
+
+        protected override async UniTask OnExecuteAsync()
+        {
+            if (Context.Parameters.Length != 1)
```
Comment (archie426) [inline NewEssentials/Commands/CTPHere.cs:38]: No need for this, OpenMod does this automatically I think

**Comment by archie426** [comment]
Apart from that, looks good, lemme grab Kr4ken quick

File: NewEssentials/Commands/CTPHere.cs (hunk 2 above)
Comment (Kr4ken-9) [reply NewEssentials/Commands/CTPHere.cs:38]: I think you might be thinking of permissions, but we can always double check

File: NewEssentials/translations.yaml (hunk 1 above)
Comment (Kr4ken-9) [reply NewEssentials/translations.yaml:140]: I will address this

**Comment by Kr4ken-9** [comment]
I have to make some changes unrelated to the /tphere command so I am going to merge and make the changes

When I saw this PR that I messed up some translations related to /god and /vanish, so those are the changes I'm referring to

File: NewEssentials/Commands/CTPHere.cs (hunk 2 above)
Comment (archie426) [reply NewEssentials/Commands/CTPHere.cs:38]: Nevermind I'm thick lol

**Pull request merged**